### ISSUE TITLE
Support FeatureUnion for InvertableHashingVectorizer

### ISSUE THREAD

**Issue by lopuhin** [TeamHG-Memex/eli5#16]
Just adding features from `.transformer_list`, possibly with prefixes, should be enough

**Comment by kmike** [comment]
Hm, do you mean it should be possible to pass FeatureUnion to InvertableHashingVectorizer, and it'll find all HashingVectorizers and apply itself to them? If so, it'd be also required to handle FeatureUnion recursively, and maybe add Pipeline support, because there can be other FeatureUnions in a FeatureUnion. I'd not put it directly to InvertableHashingVectorizer class; what about a helper function?

**Comment by lopuhin** [comment]
Yeah, it looks hard to do in general, and we also need some sensible names for the different vectorizers (parts of union) to show them in the feature report. Right now I just construct ``feature_names`` by hand, including the vectorizer name in the feature name.

I don't quite understand what the helper would do, sorry :)

**Comment by lopuhin** [comment]
Aha, I finally understand what that helper could do: we already have some rudimentary support for FeatureUnion (meaning it works out of the box in simple cases), so that helper would just build a new feature union with hashing vectorizer replaces with inverting hashing vectorizer.

**Comment by lopuhin** [comment]
FeatureUnion for text was implemented in #96, but it's still lacking unhashing support. Some work on it is in union-hashing-vec branch
